libvast/vast/format/ostream_writer.hpp

```diff
@@ -55,6 +59,33 @@ class ostream_writer : public writer {
     buf_.emplace_back(x);
   }
 
+  template <bool IncludeFieldNames, class Printer>
```

Comment:
`IncludeFieldNames` appears to be a policy. I would move it into `vast::policy`. This also helps understanding what's going on at the call site: `print<true>` vs. `print<policy::include_field_names>`